static/styles/index.css

```diff
@@ -2,69 +2,58 @@ body {
     background-image: url('../background/background.jpg');
     background-repeat: no-repeat;
     background-attachment: fixed;
-    background-size: cover;
+    background-size: 100% 100%;
     padding: 0;
     margin: 0;
 }
 
-h1 {
+#header {
```

Comment:
This makes the title very large and on my phone it ends up with 1 or 2 words per line, and even then "application" doesn't fit